src/index.js

```diff
@@ -1,46 +1,162 @@
+// IMPORT FILES
 import './reset.css';
 import './style.css';
-import dots from './images/dots.png';
-import refresh from './images/refresh.png';
-import enter from './images/enter.png';
 
-const listContainer = document.querySelector('.list');
-const enterImg = document.querySelector('#enter');
-const refreshImg = document.querySelector('#refresh');
-
-const lists = [
-  {
-    description: 'todo1',
-    completed: 'true',
-    index: '1',
-  },
+// CREATE ARRAY OF LIST OBJECTS
+let todolists = [
   {
-    description: 'todo2',
-    completed: 'false',
-    index: '2',
+    index: 1,
+    desc: 'Desc1',
+    comp: false,
   },
   {
-    description: 'todo3',
-    completed: 'true',
-    index: '3',
-  },
-  {
-    description: 'todo4',
-    completed: 'false',
-    index: '4',
+    index: 2,
+    desc: 'Desc2',
+    comp: false,
   },
 ];
 
-const showList = () => {
-  lists.forEach((i) => {
+class ToDoList {
+  constructor(index, desc, comp) {
+    this.index = index;
+    this.desc = desc;
+    this.comp = comp;
+  }
+}
+
+// LOCAL STORAGE
+class LocalStorage {
+  static getToDoLists() {
+    if (localStorage.getItem('todolists') === null) {
+      localStorage.setItem('todolists', JSON.stringify(todolists));
+    } else {
+      todolists = JSON.parse(localStorage.getItem('todolists'));
+    }
+    return todolists;
+  }
+
+  static addToDoLists(item) {
+    const todolists = LocalStorage.getToDoLists();
+    todolists.push(item);
+    localStorage.setItem('todolists', JSON.stringify(todolists));
+  }
+
+  static removeToDoLists(newdesc) {
+    const todolists = LocalStorage.getToDoLists();
+
+    todolists.forEach((item, index) => {
+      if (item.desc === newdesc) {
+        todolists.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('todolists', JSON.stringify(todolists));
+  }
+}
+
+// USER INTERFACE
+class UserInterface {
+  static showToDoLists() {
+    const todolists = LocalStorage.getToDoLists();
+
+    todolists.forEach((item) => UserInterface.addToDoLists(item));
+
+    // REMOVE TO DO LIST
+    const dustbin = document
+      .querySelector('#list')
+      .querySelectorAll('.fa-trash-can');
+
+    dustbin.forEach((bin) => {
+      bin.addEventListener('click', (e) => {
+        // Remove item from UserInterface
+        UserInterface.removeToDoLists(e.target.parentElement);
+        // Remove item from LocalStorage
+        LocalStorage.removeToDoLists(
+          e.target.parentElement.previousElementSibling.children[1].value,
+        );
+      });
+    });
+  }
+
+  static addToDoLists(item) {
```

Comment:
- [ ] After deleting all the hardcoded items in the list, it was impossible to add a new task to the array. The input field wasn't visible even after reloading the browser over and over. Kindly check out the screenshot below:
![Screenshot from 2023-01-17 23-04-17](https://user-images.githubusercontent.com/63087491/213022574-7a468ff4-2102-4884-9fd8-39fc50ff1e9d.png)

- Kindly note that it should be possible to add, delete and edit a task desc as requested in the project requirements.

Reply:
Thanks but it works fine on my end using Chrome.